src/etc/inc/globals.inc

```diff
@@ -80,8 +80,11 @@ $g = array(
 	"help_base_url" => "/help.php",
 	"pkg_prefix" => "pfSense-pkg-",
 	"default_timezone" => "Etc/UTC",
-	"language" => "en_US"
+	"language" => "en_US",
+	"default_session_timeout_mins" => 240
 );
+		// NOTE - the value of "default_session_timeout_mins" is hard-coded in guiconfig.inc, because csrf_magic is loaded early
```

Comment:
Move this comment just above default_session_timeout_mins definition, and while on that use a comment block with /* */ keeping it on 80-columns